ouroboros-consensus-cardano/src/unstable-shelley-testlib/Test/Consensus/Shelley/Examples.hs

```diff
@@ -119,6 +125,13 @@ fromShelleyLedgerExamples ShelleyLedgerExamples {
         , ("NonMyopicMemberRewards", SomeResult (GetNonMyopicMemberRewards Set.empty)
                                      (NonMyopicMemberRewards $ sreNonMyopicRewards))
         , ("GenesisConfig",          SomeResult GetGenesisConfig (compactGenesis sreShelleyGenesis))
+        , ("GetBigLedgerPeerSnapshot",
+           SomeResult GetBigLedgerPeerSnapshot
+                      (LedgerPeerSnapshot (At slotNo, [(AccPoolStake 0.9
```

Comment:
Instead of `At`, we use [`NotOrigin`](https://ouroboros-consensus.cardano.intersectmbo.org/haddocks/ouroboros-consensus/Ouroboros-Consensus-Block-Abstract.html#t:WithOrigin):
```suggestion
                      (LedgerPeerSnapshot (NotOrigin slotNo, [(AccPoolStake 0.9
```
Then, the `Cardano.Slotting.Slot` import and the new `cardano-slotting` dependency can be removed.